pest/src/position.rs

```diff
@@ -116,6 +116,10 @@ impl<'i> Position<'i> {
 
     /// Returns the line and column number of this `Position`.
     ///
+    /// This is an O(n) operation, where n is the number of lines in the input.
+    /// You better use `pair.line_col()` instead.
```

Comment:
nit: this can preferably link the corresponding rustdoc, perhaps: 
```
[`pair.line_col()`](struct.Pair.html#method.line_col)
```

or see https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html